lib/smart_answer/flow.rb

```diff
@@ -48,6 +48,20 @@ def use_session?
       ActiveModel::Type::Boolean.new.cast(@use_session)
     end
 
+    def use_escape_button(use_escape_button) # rubocop:disable Style/TrivialAccessors
+      @use_escape_button = use_escape_button
+    end
+
+    def use_escape_button?
+      return false unless use_session?
+
+      ActiveModel::Type::Boolean.new.cast(@use_escape_button)
+    end
+
+    def show_escape_link?
+      use_session? && use_escape_button?
```

Comment:
Nit: we could raise an exception when escape button is enable, but the flow doesn't use sessions. Might be less confusing for future developers - as currently fail silently and they might not understand why.

Reply:
Hi @theseanything - thanks.  Exception added and called. Test updated.